test/powershell/engine/Remoting/SessionOption.Tests.ps1

```diff
@@ -1,3 +1,4 @@
+return
```

Comment:
that was accidentally committed as those tests were failing due to external issue, will remove